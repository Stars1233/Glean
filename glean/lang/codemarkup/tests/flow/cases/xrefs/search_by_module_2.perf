{ "@generated": null, "flow.Module.3": 1, "flow.StringToFileModule.3": 1 }