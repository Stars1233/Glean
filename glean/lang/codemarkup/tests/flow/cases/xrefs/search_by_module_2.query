query: flow.SearchByModuleName { name = "imports" }
perf: True
transform: [gensort, []]
