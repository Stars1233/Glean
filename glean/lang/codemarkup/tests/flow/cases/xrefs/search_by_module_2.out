[
  "@generated",
  {
    "key": {
      "name": "imports",
      "module": { "key": { "file": { "key": "test/imports.js" } } }
    }
  }
]